client/packages/invoices/src/Returns/modals/OutboundReturn/OutboundReturn.tsx

```diff
@@ -54,6 +64,27 @@ export const OutboundReturnEditModal = ({
     }
   };
 
+  const handleNext = () => {
+    if (
+      lines.some(line => line.numberOfPacksToReturn !== 0) ||
+      zeroQuantityAlert === 'warning'
```

Comment:
I think if we're zeroing existing lines, we should call onOk here to save and close, rather than still taking the user to the blank reasons step. 

Also could you add a comment or something here that `zeroQuantityAlert === 'warning'` means we're in edit mode, and the user has already clicked OK with zero quantities once... took me a while to figure out what this was doing 😅

Reply:
> I think if we're zeroing existing lines, we should call onOk here to save and close, rather than still taking the user to the blank reasons step.

Right, yeah, good point. Will do.